### PR TITLE
PCQ-488: adding config for sscs, divorce and cmc.

### PR DESCRIPTION
**Pull request by davejones74**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PCQ-488

### Change description ###
Updated to include Divorce, CMC and SSCS configuration.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -44,6 +44,28 @@ service-config:
       case-type-ids:
         - Caveat
         - GrantOfRepresentation
+    - service: divorce
+      case-field-mappings:
+        - actor: PETITIONER
+          name: PetitionerPcqId
+        - actor: RESPONDENT
+          name: RespondantPcqId
```
Comment (Cotterale) [inline src/main/resources/application.yaml:52]: Should this be RespondentPcqId?

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -7,6 +7,8 @@
         <filePath regex="true">.*spring-security-(config|crypto|web|test|core)-5\.2\.[0-2]\.RELEASE\.jar</filePath>
         <cpe>cpe:/a:pivotal_software:spring_security</cpe>
         <cve>CVE-2018-1258</cve>
+        <cve>CVE-2020-5407</cve>
```
Comment (jackmaloney) [inline config/owasp/suppressions.xml:10]: Upgrading your spring boot should allow you to remove these suppressing FYI

File: src/main/resources/application.yaml (hunk 1 above)
Comment (davejones74) [reply src/main/resources/application.yaml]: Fixed thanks Dale.

File: config/owasp/suppressions.xml (hunk 2 above)
Comment (davejones74) [reply config/owasp/suppressions.xml:10]: Thanks Jack, spring boot upgrade is being handled in a seperate PR.

**Pull request merged**